File: go/cells-in-a-range-on-an-excel-sheet.go
func cellsInRange(s string) []string {
	startCol := int(s[0])
	startRow := int(s[1])
	endCol := int(s[3])
	endRow := int(s[4])
	output := make([]string, 0)
	for i := startCol; i <= endCol; i++ {
		for j := startRow; j <= endRow; j++ {
			col := byte(i)
			row := byte(j)
			cell := string([]byte{col, row})
			output = append(output, cell)
		}
	}
	return output
}

File: python/cells-in-a-range-on-an-excel-sheet.py
class Solution:
    def cellsInRange(self, s: str) -> List[str]:
        start_col, start_row, end_col, end_row = s[0], s[1], s[3], s[4]
        output = []
        for i in range(ord(start_col), ord(end_col) + 1):
            for j in range(ord(start_row), ord(end_row) + 1):
                col, row = chr(i), chr(j)
                cell = f"{col}{row}"
                output.append(cell)
        return output
